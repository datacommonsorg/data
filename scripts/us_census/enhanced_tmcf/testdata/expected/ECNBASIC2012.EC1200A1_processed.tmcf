Node: E:ECNBASIC2012.EC1200A1_processed->E0
typeOf: schema:Place
dcid: C:ECNBASIC2012.EC1200A1_processed->GEO_ID
name: C:ECNBASIC2012.EC1200A1_processed->NAME

Node: E:ECNBASIC2012.EC1200A1_processed->E1
typeOf: schema:StatVarObservation
name: Number of establishments
value: C:ECNBASIC2012.EC1200A1_processed->ESTAB
variableMeasured: C:ECNBASIC2012.EC1200A1_processed->new_col_0
observationAbout: E:ECNBASIC2012.EC1200A1_processed->E0
observationDate: C:ECNBASIC2012.EC1200A1_processed->YEAR

Node: E:ECNBASIC2012.EC1200A1_processed->E2
typeOf: schema:StatVarObservation
name: Sales, value of shipments, or revenue ($1,000)
value: C:ECNBASIC2012.EC1200A1_processed->RCPTOT
variableMeasured: C:ECNBASIC2012.EC1200A1_processed->new_col_1
observationAbout: E:ECNBASIC2012.EC1200A1_processed->E0
observationDate: C:ECNBASIC2012.EC1200A1_processed->YEAR
stdError: C:ECNBASIC2012.EC1200A1_processed->RCPTOT_S

Node: E:ECNBASIC2012.EC1200A1_processed->E3
typeOf: schema:StatVarObservation
name: Annual payroll ($1,000)
value: C:ECNBASIC2012.EC1200A1_processed->PAYANN
variableMeasured: C:ECNBASIC2012.EC1200A1_processed->new_col_2
observationAbout: E:ECNBASIC2012.EC1200A1_processed->E0
observationDate: C:ECNBASIC2012.EC1200A1_processed->YEAR
stdError: C:ECNBASIC2012.EC1200A1_processed->PAYANN_S

Node: E:ECNBASIC2012.EC1200A1_processed->E4
typeOf: schema:StatVarObservation
name: Number of employees
value: C:ECNBASIC2012.EC1200A1_processed->EMP
variableMeasured: C:ECNBASIC2012.EC1200A1_processed->new_col_3
observationAbout: E:ECNBASIC2012.EC1200A1_processed->E0
observationDate: C:ECNBASIC2012.EC1200A1_processed->YEAR
stdError: C:ECNBASIC2012.EC1200A1_processed->EMP_S

Node: E:ECNBASIC2012.EC1200A1_processed->E5
typeOf: schema:StatVarObservation
name: First-quarter payroll ($1,000)
value: C:ECNBASIC2012.EC1200A1_processed->PAYQTR1
variableMeasured: C:ECNBASIC2012.EC1200A1_processed->new_col_4
observationAbout: E:ECNBASIC2012.EC1200A1_processed->E0
observationDate: C:ECNBASIC2012.EC1200A1_processed->YEAR
stdError: C:ECNBASIC2012.EC1200A1_processed->PAYQTR1_S
