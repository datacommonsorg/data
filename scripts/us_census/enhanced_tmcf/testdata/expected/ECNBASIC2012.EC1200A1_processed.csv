GEO_ID,NAME,NAICS2012,TAXSTAT,TYPOP,YEAR,ESTAB,RCPTOT,PAYANN,PAYQTR1,EMP,new_col_0,new_col_1,new_col_2,new_col_3,new_col_4
country/USA,United States,21,0,0,2012,25417,529239818,59461950,N,848189,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--21&taxstat--0&typop--0
country/USA,United States,22,0,0,2012,17595,531890695,58922951,17463405,651234,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--22&taxstat--0&typop--0
country/USA,United States,23,0,0,2012,598065,1366427090,272546238,60892595,5669623,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--23&taxstat--0&typop--0
country/USA,United States,31-33,0,0,2012,297191,5696729632,593397004,N,11214165,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--31-33&taxstat--0&typop--0
country/USA,United States,42,0,0,2012,419464,7899978591,362121460,90683201,5881913,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--42&taxstat--0&typop--0
country/USA,United States,42,0,10,2012,355983,5208023478,287549597,70704319,4880666,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/EMP_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--42&taxstat--0&typop--10
country/USA,United States,42,0,20,2012,20048,2052484796,60976137,16826105,718005,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/EMP_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--42&taxstat--0&typop--20
country/USA,United States,44-45,0,0,2012,1062083,4219821871,369001350,90038805,14703529,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--44-45&taxstat--0&typop--0
country/USA,United States,48-49(104),0,0,2012,213809,730541303,183840920,45286363,4305464,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--48-49(104)&taxstat--0&typop--0
geoId/01,Alabama,21,0,0,2012,244,4722966,593200,N,8492,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--21&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--21&taxstat--0&typop--0
geoId/01,Alabama,22,0,0,2012,424,Q,1361953,454940,15454,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--22&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--22&taxstat--0&typop--0
geoId/01,Alabama,23,0,0,2012,6869,17920322,3313715,782774,78690,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--23&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--23&taxstat--0&typop--0
geoId/01,Alabama,31-33,0,0,2012,4283,124809759,11099489,N,232650,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--31-33&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--31-33&taxstat--0&typop--0
geoId/01,Alabama,42,0,0,2012,5408,75844516,3669258,901695,73312,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--42&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--42&taxstat--0&typop--0
geoId/01,Alabama,42,0,10,2012,4600,57746565,2894745,706805,60332,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/EMP_naics2012--42&taxstat--0&typop--10,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--42&taxstat--0&typop--10
geoId/01,Alabama,42,0,20,2012,282,10858345,623482,160411,9733,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/EMP_naics2012--42&taxstat--0&typop--20,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--42&taxstat--0&typop--20
geoId/01,Alabama,44-45,0,0,2012,18211,58564965,5123078,1254165,218531,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--44-45&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--44-45&taxstat--0&typop--0
geoId/01,Alabama,48-49(104),0,0,2012,2836,8422354,2460237,577477,57835,data.census.gov/table/EC1200A1/variable/ESTAB_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/RCPTOT_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYANN_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/EMP_naics2012--48-49(104)&taxstat--0&typop--0,data.census.gov/table/EC1200A1/variable/PAYQTR1_naics2012--48-49(104)&taxstat--0&typop--0
